### ISSUE TITLE
Ads on check-host.net

### ISSUE THREAD

**Issue by xglossyntx** [AdguardTeam/AdGuardExtra#512]
### Prerequisites

- [X] Filters were updated before reproducing an issue;
- [X] AdGuard Extra version is up-to-date;

### What product do you use?

AdGuard Extra with other script manager

### What type of problem have you encountered?

Missed ads or ad leftovers

### Which browser(s) do you use?

Firefox

### Where is the problem encountered?

https://check-host.net/

### What filters do you have enabled?

AdGuard Base filter, AdGuard Mobile Ads filter, AdGuard Tracking Protection filter, AdGuard URL Tracking filter, AdGuard Social Media filter, AdGuard Annoyances filter

### Add your comment and screenshots

1. The ad at the top right is not blocked. I only tested on Firefox.
https://github.com/AdguardTeam/AdguardFilters/issues/165624

<details>
<summary>Screenshot 1:</summary>

![image](https://github.com/AdguardTeam/AdGuardExtra/assets/149155649/ce60cd73-3ec7-42f0-9f34-80d469c64900)
</details>

### Privacy

- [X] I agree to follow this condition

**Comment by stanislav-atr** [comment]
i was not able to reproduce this in chrome and ff both, @jellizaveta, please, take a look

**Comment by xglossyntx** [comment]
You will need to make sure the base filter is disabled on the normal AdGuard extension, as the ads are currently blocked by it.

**Comment by xglossyntx** [comment]
Has this been fixed yet? I see it says. "Resolution: Fixed", but I'm still able to see that one ad at the right top.

**Comment by jellizaveta** [comment]
@xglossyntx Hi. Will be in beta soon.

**Comment by jellizaveta** [comment]
@xglossyntx Hello. Changes in beta version

**Comment by xglossyntx** [comment]
I just tried it, and all the ads are blocked. We can close this now if you want.

**Comment by rodrigosambadesaa** [comment]
They continue to fight back because that ad is visible again.

**Comment by xglossyntx** [comment]
For me, it's not blocking this ad.
<details>
<summary>Screenshot:</summary>

![image](https://github.com/AdguardTeam/AdGuardExtra/assets/149155649/82d69ef3-18d8-4944-bc50-a94ad6635eac)

**Comment by Samuxle** [comment]
I also have the problem that the ads come up on Twitch from Android

**Comment by xglossyntx** [comment]
> I also have the problem that the ads come up on Twitch from Android

This issue is for check-host.net. Here is the one for Twitch. https://github.com/AdguardTeam/AdGuardExtra/issues/383

**Comment by jellizaveta** [comment]
This website changes the page code frequently, which may be causing some issues. Because of this, the solution for this site is only included in the AdGuard Extra Userscript, which can be installed via Tampermonkey.

The changes are in [userscript beta version](https://userscripts.adtidy.org/beta/adguard-extra/1.0/adguard-extra.user.js), if adverts appear again please report in this issue

**Comment by xglossyntx** [comment]
I was using the beta userscript with Tamper Monkey on Firefox I had normal AdGuard disabled.

**Comment by xglossyntx** [comment]
I tried again and noticed something: some times it blocks it, other times it does not.

**Comment by jellizaveta** [comment]
No longer reproducible with the 1.0.605 version updates on my end. @zloyden could you please take a look?

**Comment by xglossyntx** [comment]
Was there a recent update? I checked yesterday, and it still did it after checking for updates, but I just pressed check updates again, and now it seems to be working.

**Comment by xglossyntx** [comment]
After pressing this now, it seems to work fine.
<details>
<summary>Screenshot:</summary>

![image](https://github.com/AdguardTeam/AdGuardExtra/assets/149155649/0b9f1e59-ee91-40a9-b338-4ace35395934)

**Comment by jellizaveta** [comment]
Yes, there's an update that came out recently. Glad to hear the adverts are hiding again, thanks for checking!

**Comment by xglossyntx** [comment]
Should I close this or just leave it open since they like to update the site so much?

**Comment by jellizaveta** [comment]
Yeah, let's keep it open in case of new updates

**Comment by rodrigosambadesaa** [comment]
They continue to fight back because that ad is visible again.

**Comment by jellizaveta** [comment]
Hi there! The solution for this site is already in beta

**Comment by zloyden** [comment]
@jellizaveta https://github.com/AdguardTeam/AdguardFilters/issues/180739 - there is an incorrect blocking that's caused by the AdGuard Extra Beta.

**Comment by jellizaveta** [comment]
Changes in beta

**Comment by rodrigosambadesaa** [comment]
Hi. I'm still seeing ads. [These](https://drive.google.com/file/d/1AWJkZNJjoJtzFLlPu_aqx5M1cVMIqL-m/view?usp=drivesdk) are my AgGuard for Android settings

**Comment by jellizaveta** [comment]
@rodrigosambadesaa Hello. As I can see, AdGuard Extra version is `1.0.600`, current version is `1.0.616`
 Could you please update AdGuard Extra userscript?

**Comment by rodrigosambadesaa** [comment]
Done, but I still see the ads.

**Comment by rodrigosambadesaa** [comment]
I just did, but I still see the ads.

**Comment by jellizaveta** [comment]
I cannot reproduce the adverts. Could you please provide more information? What browser are you using? Have you tried clearing the browser cache and refreshing the page? Is HTTPS filtering enabled for this browser? Could you also send a link to the page and screenshots of the adverts?

**Comment by xglossyntx** [comment]
I'm also unable to see the ads. I tested by using only normal AdGuard, no ads appeared. Then I tested by using only the beta userscript, and still no ads were visible.

**Comment by jellizaveta** [comment]
@rodrigosambadesaa Hello. Any news?

**Comment by rodrigosambadesaa** [comment]
It happened in Chrome for Android and in Samsung Internet. I hadn't used Samsung browser for months. After clearing data on Chrome for the site and loading it again, the ads were still visible.

**Comment by rodrigosambadesaa** [comment]
Solved the problem. HTTPS filtering was off.

**Comment by jellizaveta** [comment]
If the ads appear again — please open a new issue. I'd be happy to help!
